release-02/code/flag_tool.rb

```diff
@@ -0,0 +1,38 @@
+require 'sequel'
+
+helpText = "ITU-Minitwit Tweet Flagging Tool\n\n" +
+            "Usage:\n" +
+            "  flag_tool <tweet_id>...\n" +
+            "  flag_tool -i\n" +
+            "  flag_tool -h\n" +
+            "Options:\n" +
+            "-h            Show this screen.\n" +
+            "-i            Dump all tweets and authors to STDOUT.\n";
+
+
+def printTweet(tweet_id, user_id, tweet_text, flagged)
+    begin
+        printf "%s,%s,%s,%s\n", tweet_id, user_id, tweet_text, flagged.nil? ? 0 : flagged
+    end
+end
+
+if ARGV.length == 1 && ARGV[0].eql?("-h") then
+    puts helpText
+    exit(true)
+end
+DB = Sequel.sqlite('/tmp/minitwit.db')
+messages = DB[:message]
+
+if ARGV.length == 1 && ARGV[0].eql?("-i") then
+    for message in messages.all
+        printTweet(message[:message_id], message[:author_id], message[:text], message[:flagged])
+    end
+    exit(true)
+end
+
+if ARGV.length >= 1 && !(ARGV[0].eql? "-i") && !(ARGV[0].eql? "-h") then
+    for id in ARGV
+        messages.where(message_id: id).update(:flagged => 1);
+    end
+    exit(true)
+end
```

Comment:
I don't understand why this file is part of the pull request if it is about the refactoring of the web framework. If you want to send it to submission, I think you should open a new PR as it is important enough to be a "feature"

Reply:
Yes, good catch. I accidentally put it in there. Will remove it.